src/backend/controllers/TypeController.js

```diff
@@ -0,0 +1,46 @@
+import { Response } from "miragejs";
+
+/**
```

Comment:
Would suggest to remove comments that came with pre-installation

Reply:
These comments came with backend and help as doc.